diff --git a/node_modules/pbkdf2/lib/async.js b/node_modules/pbkdf2/lib/async.js
index 3af2c49..6c9c7f9 100644
--- a/node_modules/pbkdf2/lib/async.js
+++ b/node_modules/pbkdf2/lib/async.js
@@ -6,7 +6,7 @@ var sync = require('./sync')
 var toBuffer = require('./to-buffer')
 
 var ZERO_BUF
-var subtle = global.crypto && global.crypto.subtle
+var subtle = globalThis.crypto && globalThis.crypto.subtle
 var toBrowser = {
   sha: 'SHA-1',
   'sha-1': 'SHA-1',
@@ -20,7 +20,7 @@ var toBrowser = {
 }
 var checks = []
 function checkNative (algo) {
-  if (global.process && !global.process.browser) {
+  if (globalThis.process && !globalThis.process.browser) {
     return Promise.resolve(false)
   }
   if (!subtle || !subtle.importKey || !subtle.deriveBits) {
@@ -44,14 +44,14 @@ function getNextTick () {
   if (nextTick) {
     return nextTick
   }
-  if (global.process && global.process.nextTick) {
-    nextTick = global.process.nextTick
-  } else if (global.queueMicrotask) {
-    nextTick = global.queueMicrotask
-  } else if (global.setImmediate) {
-    nextTick = global.setImmediate
+  if (globalThis.process && globalThis.process.nextTick) {
+    nextTick = globalThis.process.nextTick
+  } else if (globalThis.queueMicrotask) {
+    nextTick = globalThis.queueMicrotask
+  } else if (globalThis.setImmediate) {
+    nextTick = globalThis.setImmediate
   } else {
-    nextTick = global.setTimeout
+    nextTick = globalThis.setTimeout
   }
   return nextTick
 }
@@ -92,7 +92,7 @@ module.exports = function (password, salt, iterations, keylen, digest, callback)
   digest = digest || 'sha1'
   var algo = toBrowser[digest.toLowerCase()]
 
-  if (!algo || typeof global.Promise !== 'function') {
+  if (!algo || typeof globalThis.Promise !== 'function') {
     getNextTick()(function () {
       var out
       try {
